tests/test_transforms/test_encoders/test_categorical_transform.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pytest
+
+from etna.datasets import TSDataset
+from etna.datasets import generate_periodic_df
+from etna.transforms.encoders.categorical import LabelBinarizerTransform
+from etna.transforms.encoders.categorical import LabelEncoderTransform
+
+
+@pytest.fixture
+def two_df_with_new_values():
+    df1 = TSDataset.to_dataset(generate_periodic_df(3, "2020-01-01", 10, 2, n_segments=2))
```

Comment:
Use named parameters here because it is not obvious what is the meaning of all these parameters.